xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/actionbuttons/actionButtons.css

```diff
@@ -25,5 +22,5 @@
 }
 
 #commentinput {
-  width: 99.7%; /* 100% - 2px border; FIXME when calc() becomes standard */
+  width: inherit;
```

Comment:
is this needed? or it can be removed?

Reply:
Yes it's needed, otherwise the width of the comment input of the inline form is changed.